storagenode/console/service.go

```diff
@@ -134,10 +135,10 @@ func (s *Service) GetDashboardData(ctx context.Context) (_ *Dashboard, err error
 	data.NodeID = s.contact.Local().Id
 	data.Wallet = s.walletAddress
 	data.Version = s.versionInfo.Version
-	data.UpToDate = s.version.IsAllowed(ctx)
 	data.StartedAt = s.startedAt
 
 	data.LastPinged = s.pingStats.WhenLastPinged()
+	data.AllowedVersion, data.UpToDate = s.version.IsAllowed(ctx)
```

Comment:
This should fail to build. You are assigning a new semver to the old struct.

Reply:
You're right. Thanks for review

Reply:
You're right. Thanks for review